common/src/java/org/apache/hive/common/util/TimestampParser.java

```diff
@@ -191,7 +190,13 @@ public Timestamp parseTimestamp(final String text)
         LOG.debug("Could not parse timestamp text: {}", text);
       }
     }
-    return Timestamp.valueOf(text);
+    Timestamp timestamp = null;
+    try {
+      timestamp = Timestamp.valueOf(text);
+    } catch (IllegalArgumentException e) {
+      LOG.info(e.getMessage());
```

Comment:
Do we need to LOG this exception? I have the impression that when this method returns null we will generate an log message anyways. Moreover, I think that logging at INFO level may be a bit too much for something that seems to be in the regular control flow.

Reply:
Removed